### DISCUSSION TITLE
funds stuck in bridge

### DISCUSSION THREAD

**Discussion by Jelyfishh** [0xPolygon/matic.js#241] [Q&A]
Hey there!

I was trying to send funds from matic to ether network, the transaction was initialized, but then it got to the stage where I have to switch to the eth network. I went with default gas fees, which turned out to be too low. Some issues happened and I canceled the transaction. But now it's still stuck in the "transaction in progress"
![Capture](https://user-images.githubusercontent.com/87905597/126872217-32075a9d-da71-4b26-8051-11f524d5b794.PNG)

Also, here is the transaction detail : https://etherscan.io/tx/0x7a2227773f728c528b8f10bff044dade076a426b18801287534eef1ffcf48b2d

How can I proceed ? 
Thanks

**Comment by ujjwalgupta94** [comment]
I see that you have completed the transaction from this tx - https://etherscan.io/tx/0x1423394334650dfe9eaac0ac3586cc471838444a89a9c1496b7b801a39baba36

Actually it takes around an hour to detect that transaction has been dropped and then we show a retry option. I hope this helps.

**Comment by ujjwalgupta94** [comment]
for any more help, please ask in our support -  https://support.polygon.technology/support/home
